doc/api/events.md

```diff
@@ -1485,11 +1485,10 @@ target.removeEventListener('foo', handler, { capture: true });
 added: v14.5.0
 -->
 
-* `event` {Object|Event}
+* `event` {Event}
```

Comment:
We could try to improve the documentation by specifying the return value:
```suggestion
* `event` {Event}
* Returns: {boolean} `true` if either event’s `cancelable` attribute value is
  false or its `preventDefault()` method was not invoked; otherwise `false`.
```
Source: https://dom.spec.whatwg.org/#dom-eventtarget-dispatchevent